File: Modul 5/PRAK504/PRAK504-2310817310011-MuhammadRizky.c
#include <stdio.h>

int reverse(int n) {
    int balik = 0;
    while (n != 0) {
        int angkaBelakang = n % 10;
        balik = balik * 10 + angkaBelakang;
        n /= 10;
    }
    return balik;
}

int main() {
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
    return 0;
}

File: Modul 5/PRAK504/PRAK504-2310817310011-MuhammadRizky.py
def reverse(n):
    balik = 0
    while n != 0:
       angkaBelakang = n % 10
       balik = balik * 10 + angkaBelakang
       n = n // 10
    return balik

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))
